<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qud Blueprint Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="tile_label">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>240</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(15, 59, 58);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_tile_button">
            <property name="text">
             <string>Save tile...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="search_label">
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="search_line_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="expand_all_button">
            <property name="text">
             <string>Expand all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="collapse_all_button">
            <property name="text">
             <string>Collapse all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restore_all_button">
            <property name="text">
             <string>Default expansion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QWidget" name="tree_target_widget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_ObjectBlueprints_xml"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View type</string>
    </property>
    <addaction name="actionWiki_template"/>
    <addaction name="actionAttributes"/>
    <addaction name="actionAll_attributes"/>
   </widget>
   <widget class="QMenu" name="menuWiki">
    <property name="title">
     <string>Wiki</string>
    </property>
    <addaction name="actionScan_wiki"/>
    <addaction name="actionUpload_templates"/>
    <addaction name="actionUpload_tiles"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWiki"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ObjectBlueprints_xml">
   <property name="text">
    <string>Open ObjectBlueprints.xml...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWiki_template">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wiki template</string>
   </property>
  </action>
  <action name="actionAttributes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Attributes</string>
   </property>
  </action>
  <action name="actionAll_attributes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All attributes</string>
   </property>
  </action>
  <action name="actionScan_wiki">
   <property name="text">
    <string>Scan wiki for selected objects</string>
   </property>
  </action>
  <action name="actionUpload_templates">
   <property name="text">
    <string>Upload templates for selected objects</string>
   </property>
  </action>
  <action name="actionUpload_tiles">
   <property name="text">
    <string>Upload tiles for selected objects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
